include/LightGBM/c_api.h

```diff
@@ -719,28 +785,29 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterPredictForCSRSingleRow(BoosterHandle handle,
 
 
 /*!
-* \brief make prediction for an new data set
-*        Note:  should pre-allocate memory for out_result,
-*               for normal and raw score: its length is equal to num_class * num_data
-*               for leaf index, its length is equal to num_class * num_data * num_iteration
-* \param handle handle
-* \param col_ptr pointer to col headers
-* \param col_ptr_type type of col_ptr, can be C_API_DTYPE_INT32 or C_API_DTYPE_INT64
-* \param indices findex
-* \param data fvalue
-* \param data_type type of data pointer, can be C_API_DTYPE_FLOAT32 or C_API_DTYPE_FLOAT64
-* \param ncol_ptr number of cols in the matrix + 1
-* \param nelem number of nonzero elements in the matrix
-* \param num_row number of rows
-* \param predict_type
-*          C_API_PREDICT_NORMAL: normal prediction, with transform (if needed)
-*          C_API_PREDICT_RAW_SCORE: raw score
-*          C_API_PREDICT_LEAF_INDEX: leaf index
-* \param num_iteration number of iteration for prediction, <= 0 means no limit
-* \param parameter Other parameters for the parameters, e.g. early stopping for prediction.
-* \param out_len len of output result
-* \param out_result used to set a pointer to array, should allocate memory before call this function
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_BoosterPredictForCSC
```

Comment:
The same comments as for `LGBM_BoosterPredictForCSR`, if other ones are not presented.